docs/reconciliation.md

```diff
@@ -73,6 +73,36 @@ annotations:
   eno.azure.io/disable-updates: "true"
 ```
 
+### Overrides
+
+Overrides let you modify specific fields of a resource during reconciliation.
+These modifications are applied on top of the synthesized resource and can be conditional.
+Conditions are CEL expressions that are evaluated against the current state of the resource at reconciliation time.
+
+This allows Eno synthesizers to specify basic runtime behavior without requiring resynthesis.
+
+> Overrides are applied during reconciliation, so in most cases they should be used alongside `eno.azure.io/reconcile-interval`.
+
+```yaml
+annotations:
+  eno.azure.io/overrides: |
+    [
+      { "path": "/data/foo", "value": "new conditional value", "condition": "self.data.bar == 'baz'" }
```

Comment:
is that possible to use self in value field? like {"path": "/data/foo", "value": "self.spec.containers[0].resources.limits.cpu"}